frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunDrawerBottomTabs.tsx

```diff
@@ -9,8 +9,8 @@ import PipelineRunTabParameters from './PipelineRunTabParameters';
 
 export enum RunDetailsTabs {
   DETAILS = 'Details',
-  PARAMETERS = 'Input parameters',
-  YAML = 'Run output',
+  PARAMETERS = 'Inputparameters',
+  YAML = 'Runoutput',
 }
 
```

Comment:
I prefer to add a enum here like
```
export enum RunDetailsTabs {
  DETAILS = 'details',
  PARAMETERS = 'input-parameters',
  YAML = 'run-output',
}

const RunDetailsTabTitles = {
  [RunDetailsTabs.DETAILS] :'Details',
  [RunDetailsTabs.PARAMETERS]: 'Input parameters',
  [RunDetailsTabs.YAML]: 'Run output',
}
```
and you only need to change title to `title={RunDetailsTabTitles[tab]}` below, which is more clear.